### PR TITLE
Add blog post about 2022 survey results

### PR DESCRIPTION
**Pull request by annajung**
Signed-off-by: Anna Jung (VMware) <antheaj@vmware.com>

Add blog post about 2022 Kubefow user survey results

### CODE REVIEW

**Comment by annajung** [comment]
deploy preview: https://deploy-preview-121--infallible-murdock-d05108.netlify.app/kubeflow-user-survey-2022/

**Comment by annajung** [comment]
cc @kubeflow/release-team @jbottum @DomFleischmann

File: _posts/2022-08-19-kubeflow-user-survey-2022.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  <figcaption>2021</figcaption>
+</figure>
+
+## Full Survey Results
+
+Full survey results can be found in the [initial survey results](https://groups.google.com/u/2/g/kubeflow-discuss/c/UXNmOWNmu1k) and in the [summary of the free-from answers](https://docs.google.com/document/d/1TKI-KR4oopm0G6oIMwUUJ_z6ox5BE6fnjSbEUoisnLE/edit#heading=h.e8zqwmv6lumd).
+
+## What’s Next?
+
+Survey results will be discussed with the Kubeflow community during the Kubeflow summit in October. More details about the summit will be shared via the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list. Join the mailing list to keep up to date with Kubeflow summit news.
```
Comment (annajung) [inline _posts/2022-08-19-kubeflow-user-survey-2022.md:127]: @akgraner Let me know if you like any changes here about the kubeflow summit

**Comment by annajung** [comment]
Hi @james-jwu @zijianjoy @theadactyl, could you take a look? I think it would be helpful to get this published before the upcoming Kubeflow summit so that everyone has a pointer to discuss survey results. 

cc @akgraner @jbottum

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you for composing the blogpost!

File: _posts/2022-08-19-kubeflow-user-survey-2022.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+author: "Kubeflow 1.6 Release Team"
+---
+
+In May 2022, the Kubeflow user survey was opened to gather community feedback. The goal of the survey was to understand the adoption of Kubeflow and to gather input on the benefits, gaps, and requirements for machine learning use cases.
+
+The survey was comprised of 24 questions (multiple choice and freeform). It ran from May 6th to June 7th and received 151 responses. Out of the 151 responses, 91 responses provided optional feedback on how to improve Kubeflow and the Community. In the 2022 survey, we received an increased number of freeform answers, which provided additional insights into users’ needs.
+
+## Key Takeaways
+
+- 85% of the user deploy more than one Kubeflow component
```
Comment (jbottum) [inline _posts/2022-08-19-kubeflow-user-survey-2022.md:19]: I think user should be plural i.e. 85% of the users deploy more than one Kubeflow component

File: _posts/2022-08-19-kubeflow-user-survey-2022.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-user-survey-2022/
+author: "Kubeflow 1.6 Release Team"
+---
+
+In May 2002, the Kubeflow user survey was open to gather community feedback. The goal of the survey was to understand the adoption of Kubeflow and to gather input on the benefits, gaps, and requirements for machine learning use cases.
```
Comment (jbottum) [inline _posts/2022-08-19-kubeflow-user-survey-2022.md]: May 2022

Comment (annajung) [reply _posts/2022-08-19-kubeflow-user-survey-2022.md]: Done

File: _posts/2022-08-19-kubeflow-user-survey-2022.md (hunk 2 above)
Comment (annajung) [reply _posts/2022-08-19-kubeflow-user-survey-2022.md]: Done

**Pull request merged**